examples/external/swap_amber_parameters/swap_existing_ligand_parameters.ipynb

```diff
@@ -18,14 +18,7 @@
   {
```

Comment:
<p>The description was upgraded to 2.1.0 but the code still points to 2.0.0</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-toolkit/pull/1719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>